tests/unit_tests/test_line.py

```diff
@@ -0,0 +1,66 @@
+from SyntheticDataset.image_operations import Line
+from SyntheticDataset.image_operations import Point
+import unittest
+from math import sqrt
+
+class LineTestCase(unittest.TestCase):
+
+    def test_set_slope(self):
+        L = Line(Point(1,2),Point(5,10))
+        L.setSlope()
+        self.assertEqual(2, L.slope)
+
+    def test_get_Y_as_function(self):
+        L= Line(Point(1,2),Point(5,10))
```

Comment:
The variable name needs to be better, as well as please fix the spacing around the ```=``` operator.